app/pages/membership-stripe.vue

```diff
@@ -16,27 +16,25 @@
           </div>
 
           <div>
-            <form action="https://cms.lahmacun.hu/wp-json/stripe/return_checkout_session_recurring_membership"
-              method="GET">
-              <div>
-                <label for="show_name">Select your show</label>
-                <div class="relative max-w-md bg-white rounded-sm">
-                  <select name="show_name" class="show-select" :class="{ showSelected: show_name.length > 1 }" @change="selectShow">
-                    <option disabled selected value="">Please select a show</option>
-                    <option v-for="(show) in arcsiShowsList" :value="show.name"
-                      :key="show.archive_lahmastore_base_url">
-                      {{ show.name }}
-                    </option>
-                  </select>
-                  <div class="absolute right-2 top-2 z-0">
-                    <i class="fa fa-chevron-down" />
-                  </div>
-                </div>
+            <form :action="$config.membershipStripeFormUrl" method="GET">
+              <div class="selector">
+
+                <v-app>
+                  <v-container>
+                    <v-select label="Select your show" name="show_name" v-model="show_name"
+                      :items="arcsiShowsList"></v-select>
+                  </v-container>
+                </v-app>
+                <input type="hidden" name="show_name">
```

Comment:
yes, is the url correct? because the selector works fine. I added a fix also for the value now

Reply:
Yes, now I realized, that there was also an issue with the Stripe secret key and the configured URL on the CMS/"backend" side.
I resolved that misconfiguration and it works now, but this duplicated `show_name` parameter still looks a bit odd for me, even both of them has the same value.
I tested it manually on the dev environment and if you remove [this line](https://github.com/lahmacunradio/frontend/pull/308/files#) it will work fine.

Reply:
Oh, OK, I thought the v-model was only making the local state and not the form params, thanks, fixed it now